nestle.py

```diff
@@ -65,6 +65,34 @@ def random_choice(a, p, rstate=np.random):
         t += p[i]
     return i
 
+def systematic_resample(weights):
+    """
+    Return the resampling indices using the weights. It uses systematic resampling
+    Based on http://users.isy.liu.se/rt/schon/Publications/HolSG2006.pdf
+    
+    Parameters
+    ----------
+    weights : weight of each sample
```

Comment:
Format docstring parameters and return values like
```
weights : `~numpy.ndarray`
    Weight of each sample.
```
This follows the numpydoc format which allows the auto-generated documentation tools to work to create [the API docs](http://kbarbary.github.io/nestle/api.html)